src/lib/utils.ts

```diff
@@ -1,3 +1,6 @@
+import variantMappings from '../../data/variant-mapping.json'
```

Comment:
Minor: It'd be nice to be consistent here on the naming in terms of singular vs plural. E.g one of:

* `variantMappings` and `variant-mappings.json` 
* `variantMapping` and `variant-mapping.json`

I'd maybe lean towards the former and then `getVariantMapping` as the name for the function which identifies the appropriate mapping from `variantMappings` :man_shrugging: .